examples/blinky-stm32h747-disco.rs

```diff
@@ -0,0 +1,67 @@
+//! Basic example that produces a 1Hz square-wave on Pin PE1
+ 
+//#![deny(warnings)] // This code runs on stm32h747i-disco and does not use example! macro.
```

Comment:
Please keep the deny(warnings), but you can use an allow(unused_macros)

```suggestion
#![deny(warnings)] 
#![allow(unused_macros)] // This code runs on stm32h747i-disco and does not use example! macro.
```